source/installation_guide/dovecot_pro_releases/index.rst

```diff
@@ -1,4 +1,4 @@
-.. _ox_dovecot_pro_releases:
+.. _ox_dovecot_pro:
   
 #######################
 OX Dovecot Pro Releases
```

Comment:
The main thinking I had with the label change was that the title would also point to "OX Dovecot Pro". I guess maybe here this could simply be changed to remove the "Releases" part. Then the simplification is mainly that instead of:

```
 :ref:`OX Dovecot Pro <ox_dovecot_pro>`
```

You can just have:

```
 :ref:`ox_dovecot_pro`
```

I don't know if we should have had separate labels for ox_dovecot_pro_releases pointing to "OX Dovecot Pro Releases" and one with ox_dovecot_pro pointing to "OX Dovecot Pro", but .. I suppose there is no need for such distinction.